Tensile/LibraryIO.py

```diff
@@ -132,7 +132,7 @@ def readLibraryLogicForSchedule( filename ):
 
   # verify
   if len(data) < 6:
-    printExit("len(%s) %u < 7" % (filename, len(data)))
+    printExit("len(%s) %u < 6" % (filename, len(data)))
```

Comment:
What does '6' or '7' mean? Don't have enough data?
Can we improve this with a more verbose message?

Reply:
Initially I just changed the message to match the condition without looking any further into it. Upon further inspection, this condition hasn't been updated in ~4 years; I'll update it to be current and improve the message.